gameOOP.js

```diff
@@ -345,41 +345,48 @@ class Player extends Character {
     }
 
     getKey(event) {
-        if (event.code === "KeyW") {
-            this.move(0, -1);
-
-        } else if (event.code === "KeyS") {
-            this.move(0, 1);
-        }
-
-        if (event.code === "KeyD") {
-            this.move(1);
-
-        } else if (event.code === "KeyA") {
-            this.move(-1);
-
-        }
+        // Note 4: This could be an entire class with the rendering of menues
+        // Close all menues unless is the one being toggle, menu key control should be outside the character
+        let openMenu = (menues, menu) => {

```

Comment:
This could be an entire class with the rendering of menues
Close all menues unless is the one being toggle, menu key control should be outside the character